docs/data/joy/components/drawer/drawer.md

```diff
@@ -7,64 +7,66 @@ githubLabel: 'component: drawer'
 
 # Drawer
 
-<p class="description">Navigation drawers provide quick access to other destinations in your app without removing the user out of context.</p>
+<p class="description">Navigation drawers provide quick access to other app areas without removing the user out of context.</p>
 
 ## Introduction
 
+The Drawer component is commonly used as a type of menu on desktop and as a dialog on mobile (similar to [Apple's sheets](https://developer.apple.com/design/human-interface-guidelines/sheets)).
+
 {{"demo": "DrawerUsage.js", "hideToolbar": true, "bg": "gradient"}}
 
 ## Basic
 
-The navigation drawers can toggle open or closed. Closed by default, the drawer opens temporarily above all other content until a section is selected.
-
-The Drawer can be cancelled by clicking the overlay or pressing the Esc key.
-It closes when an item is selected, handled by controlling the `open` prop.
+The Drawer can be closed by clicking on the overlay or pressing the <kbd>Esc</kbd> key.
+It will also close if an item inside of it is selected, which can be controlled by the `open` prop.
 
 {{"demo": "DrawerBasic.js"}}
 
 ## Customization
 
 ### Anchor
```

Comment:
@siriwatknp It seems the default anchor value is `left`, right? Shouldn't we mention it as well as add a Playground control for it?

Reply:
Yes, we should mention it. We can move the toggle button to an anchor in the playground and have just one button.

Reply:
Agreed, yeah! That'd communicate what is the default one and allow visitors to see what other positions look like!
Let me know if I can help with anything as far as doing it!

Reply:
@siriwatknp Can you help out with this one? 😬

Reply:
Thank you! 🎉 🙌